File: client/lib/models/Transaction.dart
class Transaction {
  int id;
  double creditAmount;
  double debitAmount;
  double amount;
  int accountHolderId;
  int ledgerId;
  String ledgerName;
  String transactionId;
  String accountNumber;
  String paymentGatewayName;
  String createdAt;
  String status;

  Transaction({
    this.id,
    this.creditAmount,
    this.debitAmount,
    this.amount,
    this.accountHolderId,
    this.ledgerId,
    this.ledgerName,
    this.transactionId,
    this.accountNumber,
    this.paymentGatewayName,
    this.createdAt,
    this.status
  });
}

File: server/app/Models/Transaction.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;


class Transaction extends Model
{

    protected $table = 'Transactions';
    public $timestamps = false;

    protected $fillable = [
        'id',
        'creditAmount', // deposit 101
        'debitAmount', // withdraw 102
        'accountHolderId',
        'ledgerId',
        'transactionId',
        'accountNumber',
        'paymentGatewayName',
        'status',
        'ip',
        'modifiedBy',
        'createdAt'
    ];
}
